src/PersonObjects/Player/PlayerObjectGeneralMethods.ts

```diff
@@ -548,22 +549,16 @@ export function reapplyAllAugmentations(this: PlayerObject, resetMultipliers = t
     this.resetMultipliers();
   }
 
-  for (let i = 0; i < this.augmentations.length; ++i) {
-    //Compatibility with new version
-    if (this.augmentations[i].name === "HacknetNode NIC Architecture Neural-Upload") {
```

Comment:
We don't still need this upgrade code somewhere?

Reply:
Ah I had initially written that on a different branch where I had more safe load code (the other branch was converting person.augmentations to JSONMap, it ended up being way too huge before the first commit). Looks like I will need to re-add safe load code in (will just splice out any augs with invalid names)

Reply:
Added safe load code to filter out invalid augs.